### PR TITLE
chore: tweaks to verbiage of README

### PR DESCRIPTION
**Pull request by jamesonwilliams**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,45 +7,33 @@
...
-- **[Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/android)**
-  Easily collect analytics data for your app. Analytics data includes user
-  sessions and other custom events.
-- **[Predictions](https://docs.amplify.aws/lib/predictions/getting-started/q/platform/android)**
-  Connect your application with machine learning cloud services to enhance your
-  application with natural language processing, computer vision, text to speech,
-  and more.
+| Category       | AWS Provider | Description                                |
+|----------------|--------------|--------------------------------------------|
+| Authentication | Cognito      | Building blocks to create auth experiences |
```
Comment (richardmcclellan) [inline README.md:30]: Can we make the category titles link to the docs, like they do currently?

Comment (jamesonwilliams) [reply README.md:30]: Oh, der. Excellent idea. Yes, I will update.

File: README.md [hunk 2]
```diff
@@ -7,45 +7,33 @@
...
+| Storage        | S3           | Manages content in public, protected, private storage buckets |
+| DataStore      | AppSync      | Programming model for shared and distributed data, with simple online/offline syncrhonization |
```
Comment (richardmcclellan) [inline README.md:32]:
```suggestion
| DataStore      | AppSync      | Programming model for shared and distributed data, with simple online/offline synchronization |
```

File: README.md [hunk 3]
```diff
@@ -7,45 +7,33 @@
...
+| API (GraphQL)  | AppSync      | Interact with your GraphQL or AppSync endpoint |
+| API (REST)     | API Gateway  | Sig4 signing and AWS auth for API Gateway and other REST endpoints |
```
Comment (richardmcclellan) [inline README.md:34]:
```suggestion
| API (REST)     | API Gateway  | Sigv4 signing and AWS auth for API Gateway and other REST endpoints |
```

**Review by richardmcclellan — APPROVED** [review]
Minor comments, but overall looks good! 🚀

**Pull request merged**